runtime/libpgmath/lib/common/mthdecls.h

```diff
@@ -193,6 +193,8 @@ quad_complex_t pgmath_cmplxl(float128_t r, float128_t i)
         float _f(float real, float imag)
 #define	DBLFUNC_C_(_f)    \
         double _f(double real, double imag)
+#define	QUADFUNC_C_(_f)    \
+        long double _f(long double real, long double imag)
```

Comment:
Use `float128_t` instead of `long double`, which is not portable.

Reply:
OK, already updated.